File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/ciordems/d/EmsOutItemDO.java
package iih.ci.ord.ciordems.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 出院医嘱申请单 DTO数据 
 * 
 */
public class EmsOutItemDO extends OrCommonFieldsDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 主键
	 * @return String
	 */
	public String getId_emsapout() {
		return ((String) getAttrVal("Id_emsapout"));
	}
	/**
	 * 主键
	 * @param Id_emsapout
	 */
	public void setId_emsapout(String Id_emsapout) {
		setAttrVal("Id_emsapout", Id_emsapout);
	}
	/**
	 * 医嘱id
	 * @return String
	 */
	public String getId_or() {
		return ((String) getAttrVal("Id_or"));
	}
	/**
	 * 医嘱id
	 * @param Id_or
	 */
	public void setId_or(String Id_or) {
		setAttrVal("Id_or", Id_or);
	}
	/**
	 * 医嘱服务id
	 * @return String
	 */
	public String getId_orsrv() {
		return ((String) getAttrVal("Id_orsrv"));
	}
	/**
	 * 医嘱服务id
	 * @param Id_orsrv
	 */
	public void setId_orsrv(String Id_orsrv) {
		setAttrVal("Id_orsrv", Id_orsrv);
	}
	/**
	 * 出院科室id
	 * @return String
	 */
	public String getId_dep_out() {
		return ((String) getAttrVal("Id_dep_out"));
	}
	/**
	 * 出院科室id
	 * @param Id_dep_out
	 */
	public void setId_dep_out(String Id_dep_out) {
		setAttrVal("Id_dep_out", Id_dep_out);
	}
	/**
	 * 出院科室
	 * @return String
	 */
	public String getName_dep_out() {
		return ((String) getAttrVal("Name_dep_out"));
	}
	/**
	 * 出院科室
	 * @param Name_dep_out
	 */
	public void setName_dep_out(String Name_dep_out) {
		setAttrVal("Name_dep_out", Name_dep_out);
	}
	/**
	 * 出院病区id
	 * @return String
	 */
	public String getId_dep_nur_out() {
		return ((String) getAttrVal("Id_dep_nur_out"));
	}
	/**
	 * 出院病区id
	 * @param Id_dep_nur_out
	 */
	public void setId_dep_nur_out(String Id_dep_nur_out) {
		setAttrVal("Id_dep_nur_out", Id_dep_nur_out);
	}
	/**
	 * 出院病区
	 * @return String
	 */
	public String getName_dep_nur_out() {
		return ((String) getAttrVal("Name_dep_nur_out"));
	}
	/**
	 * 出院病区
	 * @param Name_dep_nur_out
	 */
	public void setName_dep_nur_out(String Name_dep_nur_out) {
		setAttrVal("Name_dep_nur_out", Name_dep_nur_out);
	}
	/**
	 * 出院床位id
	 * @return String
	 */
	public String getId_bed_out() {
		return ((String) getAttrVal("Id_bed_out"));
	}
	/**
	 * 出院床位id
	 * @param Id_bed_out
	 */
	public void setId_bed_out(String Id_bed_out) {
		setAttrVal("Id_bed_out", Id_bed_out);
	}
	/**
	 * 出院床位
	 * @return String
	 */
	public String getName_bde_out() {
		return ((String) getAttrVal("Name_bde_out"));
	}
	/**
	 * 出院床位
	 * @param Name_bde_out
	 */
	public void setName_bde_out(String Name_bde_out) {
		setAttrVal("Name_bde_out", Name_bde_out);
	}
	/**
	 * 31日再入院计划
	 * @return FBoolean
	 */
	public FBoolean getFg_again31() {
		return ((FBoolean) getAttrVal("Fg_again31"));
	}
	/**
	 * 31日再入院计划
	 * @param Fg_again31
	 */
	public void setFg_again31(FBoolean Fg_again31) {
		setAttrVal("Fg_again31", Fg_again31);
	}
	/**
	 * 31日再入院目的
	 * @return String
	 */
	public String getDes_again31() {
		return ((String) getAttrVal("Des_again31"));
	}
	/**
	 * 31日再入院目的
	 * @param Des_again31
	 */
	public void setDes_again31(String Des_again31) {
		setAttrVal("Des_again31", Des_again31);
	}
	/**
	 * 离院方式id
	 * @return String
	 */
	public String getId_outtp() {
		return ((String) getAttrVal("Id_outtp"));
	}
	/**
	 * 离院方式id
	 * @param Id_outtp
	 */
	public void setId_outtp(String Id_outtp) {
		setAttrVal("Id_outtp", Id_outtp);
	}
	/**
	 * 离院方式编码
	 * @return String
	 */
	public String getSd_outtp() {
		return ((String) getAttrVal("Sd_outtp"));
	}
	/**
	 * 离院方式编码
	 * @param Sd_outtp
	 */
	public void setSd_outtp(String Sd_outtp) {
		setAttrVal("Sd_outtp", Sd_outtp);
	}
	/**
	 * 离院方式
	 * @return String
	 */
	public String getName_outtp() {
		return ((String) getAttrVal("Name_outtp"));
	}
	/**
	 * 离院方式
	 * @param Name_outtp
	 */
	public void setName_outtp(String Name_outtp) {
		setAttrVal("Name_outtp", Name_outtp);
	}
	/**
	 * 离院描述
	 * @return String
	 */
	public String getDes_outtp() {
		return ((String) getAttrVal("Des_outtp"));
	}
	/**
	 * 离院描述
	 * @param Des_outtp
	 */
	public void setDes_outtp(String Des_outtp) {
		setAttrVal("Des_outtp", Des_outtp);
	}
	/**
	 * sv
	 * @return FDateTime
	 */
	public FDateTime getSv() {
		return ((FDateTime) getAttrVal("Sv"));
	}
	/**
	 * sv
	 * @param Sv
	 */
	public void setSv(FDateTime Sv) {
		setAttrVal("Sv", Sv);
	}
	/**
	 * 出院时间
	 * @return FDateTime
	 */
	public FDateTime getDt_out() {
		return ((FDateTime) getAttrVal("Dt_out"));
	}
	/**
	 * 出院时间
	 * @param Dt_out
	 */
	public void setDt_out(FDateTime Dt_out) {
		setAttrVal("Dt_out", Dt_out);
	}
	/**
	 * 是否死亡标识
	 * @return FBoolean
	 */
	public FBoolean getFg_death() {
		return ((FBoolean) getAttrVal("Fg_death"));
	}
	/**
	 *  是否死亡标识
	 * @param Fg_death
	 */
	public void setFg_death(FBoolean Fg_death) {
		setAttrVal("Fg_death", Fg_death);
	}
	/**
	 * 是否尸检标识
	 * @return FBoolean
	 */
	public FBoolean getFg_autopsy() {
		return ((FBoolean) getAttrVal("Fg_autopsy"));
	}
	/**
	 *  是否尸检标识
	 * @param Fg_autopsy
	 */
	public void setFg_autopsy(FBoolean Fg_autopsy) {
		setAttrVal("Fg_autopsy", Fg_autopsy);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.ord/ciordems/d/EmsOutItemDO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;
using iih.ci.iih.ci.ord.ciordems.d;

namespace iih.ci.ord.ciordems.d
{
    /// <summary>
    /// EmsOutItemDO 的摘要说明。
    /// </summary>
    public class EmsOutItemDO : OrCommonFieldsDTO{

        public EmsOutItemDO() {
 
        }

        /// <summary>
        /// 主键
        /// </summary>
		public string Id_emsapout {
            get { return getAttrVal<string>("Id_emsapout",null); }
            set { setAttrVal<string>("Id_emsapout", value); }
        }

        /// <summary>
        /// displaynam14
        /// </summary>
		public string Id_or {
            get { return getAttrVal<string>("Id_or",null); }
            set { setAttrVal<string>("Id_or", value); }
        }

        /// <summary>
        /// displaynam15
        /// </summary>
		public string Id_orsrv {
            get { return getAttrVal<string>("Id_orsrv",null); }
            set { setAttrVal<string>("Id_orsrv", value); }
        }

        /// <summary>
        /// 出院科室id
        /// </summary>
		public string Id_dep_out {
            get { return getAttrVal<string>("Id_dep_out",null); }
            set { setAttrVal<string>("Id_dep_out", value); }
        }

        /// <summary>
        /// 出院科室
        /// </summary>
		public string Name_dep_out {
            get { return getAttrVal<string>("Name_dep_out",null); }
            set { setAttrVal<string>("Name_dep_out", value); }
        }

        /// <summary>
        /// 出院病区id
        /// </summary>
		public string Id_dep_nur_out {
            get { return getAttrVal<string>("Id_dep_nur_out",null); }
            set { setAttrVal<string>("Id_dep_nur_out", value); }
        }

        /// <summary>
        /// 出院病区
        /// </summary>
		public string Name_dep_nur_out {
            get { return getAttrVal<string>("Name_dep_nur_out",null); }
            set { setAttrVal<string>("Name_dep_nur_out", value); }
        }

        /// <summary>
        /// 出院床位id
        /// </summary>
		public string Id_bed_out {
            get { return getAttrVal<string>("Id_bed_out",null); }
            set { setAttrVal<string>("Id_bed_out", value); }
        }

        /// <summary>
        /// 出院床位
        /// </summary>
		public string Name_bde_out {
            get { return getAttrVal<string>("Name_bde_out",null); }
            set { setAttrVal<string>("Name_bde_out", value); }
        }

        /// <summary>
        /// 31日再入院计划
        /// </summary>
		public bool? Fg_again31 {
            get { return getAttrVal<FBoolean>("Fg_again31",null); }
            set { setAttrVal<FBoolean>("Fg_again31", value); }
        }

        /// <summary>
        /// 31日再入院目的
        /// </summary>
		public string Des_again31 {
            get { return getAttrVal<string>("Des_again31",null); }
            set { setAttrVal<string>("Des_again31", value); }
        }

        /// <summary>
        /// 离院方式id
        /// </summary>
		public string Id_outtp {
            get { return getAttrVal<string>("Id_outtp",null); }
            set { setAttrVal<string>("Id_outtp", value); }
        }

        /// <summary>
        /// 离院方式编码
        /// </summary>
		public string Sd_outtp {
            get { return getAttrVal<string>("Sd_outtp",null); }
            set { setAttrVal<string>("Sd_outtp", value); }
        }

        /// <summary>
        /// 离院方式
        /// </summary>
		public string Name_outtp {
            get { return getAttrVal<string>("Name_outtp",null); }
            set { setAttrVal<string>("Name_outtp", value); }
        }

        /// <summary>
        /// 离院描述
        /// </summary>
		public string Des_outtp {
            get { return getAttrVal<string>("Des_outtp",null); }
            set { setAttrVal<string>("Des_outtp", value); }
        }

        /// <summary>
        /// 服务内部编码
        /// </summary>
        public string Innercode_srvca
        {
            get { return getAttrVal<string>("Innercode_srvca", null); }
            set { setAttrVal<string>("Innercode_srvca", value); }
        }


        /// <summary>
        /// 费用同步标识
        /// </summary>
        public bool? Fg_syncfee
        {
            get { return getAttrVal<bool?>("Fg_syncfee", null); }
            set { setAttrVal<bool?>("Fg_syncfee", value); }
        }

        /// <summary>
        /// sv
        /// </summary>
        public DateTime? Sv
        {
            get { return getAttrVal<FDateTime>("Sv", null); }
            set { setAttrVal<FDateTime>("Sv", value); }
        }

        /// <summary>
        /// 出院时间
        /// </summary>
        public DateTime? Dt_out
        {
            get { return getAttrVal<FDateTime>("Dt_out", null); }
            set { setAttrVal<FDateTime>("Dt_out", value); }
        }

        /// <summary>
        /// 是否死亡
        /// </summary>
        public bool? Fg_death
        {
            get { return getAttrVal<FBoolean>("Fg_death", false); }
            set { setAttrVal<FBoolean>("Fg_death", value); }
        }

        /// <summary>
        /// 死亡尸检标识
        /// </summary>
        public bool? Fg_autopsy
        {
            get { return getAttrVal<FBoolean>("Fg_autopsy", false); }
            set { setAttrVal<FBoolean>("Fg_autopsy", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[] { "Id_emsapout", "Id_or", "Id_orsrv", "Id_dep_out", "Name_dep_out", "Id_dep_nur_out", "Name_dep_nur_out", "Id_bed_out", "Name_bde_out", "Fg_again31", "Des_again31", "Id_outtp", "Sd_outtp", "Name_outtp", "Des_outtp", "Sv", "Dt_out", "Fg_death", "Fg_autopsy" };
        }

        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.ciordems.d.EmsOutItemDO";
        }
    }
}
